cfy_manager/components/validations.py

```diff
@@ -90,6 +91,45 @@ def _validate_supported_distros():
         )
 
 
+def _validate_private_ip():
+    logger.info('Validating private IP address...')
+    private_ip = config[MANAGER][PRIVATE_IP]
+    all_addresses = set()
+    found = False
+    all_interfaces = netifaces.interfaces()
+    for interface in all_interfaces:
+        int_addresses = netifaces.ifaddresses(interface)
+        if not int_addresses:
+            logger.debug('Could not find any addresses for interface {}'.
+                format(interface))
+            continue
+        inet_addresses = int_addresses.get(netifaces.AF_INET)
+        if not inet_addresses:
+            logger.debug('No AF_INET addresses found for interface {}'.format(
+                interface))
+            continue
+        for inet_addr in inet_addresses:
+            addr = inet_addr.get('addr')
+            if not addr:
+                logger.debug('No addr found for {}'.format(inet_addr))
+                continue
+            if addr == private_ip:
+                found = True
+                break
+            all_addresses.add(addr)
+        if found:
+            break
+
+    if not found:
+        _errors.append(
+            'The provided private IP ({}) is not associated with any INET-'
+            'type address available on this machine (available INET-type '
+            'addresses: {}). This will cause installation to fail. If you '
+            'are certain that this IP address should be used, please set '
+            'the \'{}\' parameter to \'false\''.format(
```

Comment:
Let's do named params here, as this might be tougher to follow. e.g. 
```
'the {skip_validations} parameter'.format(skip_validations=SKIP_VALIDATIONS)
```